pkl-commons-cli/src/main/kotlin/org/pkl/commons/cli/commands/BaseOptions.kt

```diff
@@ -298,13 +298,13 @@ class BaseOptions : OptionGroup() {
   val traceMode: TraceMode by
     option(
         names = arrayOf("--trace-mode"),
-        metavar = "when",
+        metavar = "style",
         help =
-          "Specifies how calls to trace() are formatted. Possible values of <when> are 'default', 'pretty' and 'hidden'.",
+          "Specifies how calls to trace() are formatted. Possible values of <style> are 'compact' and 'pretty'.",
       )
       .enum<TraceMode> { it.name.lowercase() }
       .single()
-      .default(TraceMode.DEFAULT)
+      .default(TraceMode.COMPACT)
```

Comment:
This should be nullable; we want this to fall back to the setting in `pkl:Project` if available.

Take a look at `org.pkl.commons.cli.CliCommand` and see how many of those options are set, and how the evaluator is built based on this.

This should also fix the broken tests on main currently.